lms/djangoapps/certificates/api.py

```diff
@@ -540,7 +540,12 @@ def _get_two_letter_language_code(language_code):
     Shortens language to only first two characters (e.g. es-419 becomes es)
     This is needed because Catalog returns locale language which is not always a 2 letter code.
     """
-    return language_code[:2] if language_code else None
+    if language_code is None:
+        return None
+    elif language_code == '':
+        return ''
+    else:
+        return language_code[:2]
```

Comment:
@McKenzieW Could you please verify that this expresses the correct behavior for certificate lookup?

Reply:
👍